scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -42,10 +49,11 @@ def cpuflags():
 
 
 def hypervisor_has_virtio(libvirt_type):
-    if (libvirt_type == "xen" or libvirt_type == "hyperv"):
-        return False
-    else:
+
+    if libvirt_type == "kvm":
```

Comment:
yes, only kvm has virtio, so should be good. (also KVM would work with the non-virtio mode as well)